src/components/ViewArticle.jsx

```diff
@@ -26,6 +26,10 @@ export default function ViewArticle(props) {
         })
     }, [])
 
+    function handleVote() {
```

Comment:
this function just calls another function and does nothing else - consider moving the logic for voteOnArticle to here instead to remove the layer in between